lightning/src/ln/chan_utils.rs

```diff
@@ -217,87 +217,100 @@ pub fn derive_public_key<T: secp256k1::Signing>(secp_ctx: &Secp256k1<T>, per_com
 
 /// Derives a per-commitment-transaction revocation key from its constituent parts.
 ///
+/// Only the transaction broadcaster owns a valid witness to propagate a revoked
```

Comment:
s/revoked/revocable/ plus in the comment below.

Reply:
Not gonna to take it, to avoid modifying commits.

Reply:
Wait, isn't this fully opposite, though - only the countersignatory of the originally-broadcasted commitment transaction can create a valid witness to broadcast a revocation key? I suppose its now a little confused - the commitment-tx-countersignatory is now the broadcaster of the revocation-punishment transaction (hence the per_commitment_secret is sent). I don't know that this needs to be captured in the names per se, but its probably worth at least fixing the comment (a new commit is fine by me, though).